src/longbow/segment/command.py

```diff
@@ -216,8 +216,10 @@ def _sub_process_work_fn(in_queue, out_queue):
             return
 
         # Unpack our data here:
+        read = raw_data
```

Comment:
This doesn't seem to do anything - probably faster to just use `raw_data` in the `fromstring` construction below.

Reply:
Good point, will change.